### PR TITLE
*(ticdc): do not print password in cdc log

### LINKED ISSUE #9690: do not print password in cdc log
Do not print passwords in cdc log

### PR DESCRIPTION
**Pull request by sdojjy**
### What problem does this PR solve?

Issue Number: close #9690

### What is changed and how it works?
1. mask password
2. do not print configurations

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
`None`.
```

### CODE REVIEW

File: cdc/api/v2/changefeed.go [hunk 1]
```diff
@@ -176,7 +170,7 @@ func (h *OpenAPIV2) createChangefeed(c *gin.Context) {
 
 	log.Info("Create changefeed successfully!",
 		zap.String("id", info.ID),
-		zap.String("changefeed", infoStr))
+		zap.String("changefeed", info.String()))
```
Comment (asddongmen) [inline cdc/api/v2/changefeed.go:173]: `changefeedInfo.String()` should be modify to mask all possible credentials information.

Comment (sdojjy) [reply cdc/api/v2/changefeed.go:173]: fixed

**Review by asddongmen — APPROVED** [review]

**Review by nongfushanquan — APPROVED** [review]

**Pull request merged**